GettingStarted/CheatSheet.txt

```diff
@@ -0,0 +1,155 @@
+
+
+-----------------------------------------------------------
+
+Connecting in to the LSST site at NCSA...
+
+Start CiscoAnyconnect Client for NCSA VPN.  Use NCSA kerberos principal and password for the
+NCSA VPN username and password, and the 6-digit number provided by DUO for the second password. 
+
+JupyterLab:
+
+  Point broswer to https://lsst-lspdev.ncsa.illinois.edu/nb
+
+  Release version, Medium --> Spawn
+  +Launcher --> terminal
+
+  cd notebooks
+  (first time:)  git clone https://github.com/LSSTScienceCollaborations/StackClub.git
+  cd StackClub
+  git remote -v
+  git branch -a    # list all branches
+  git status       # very useful; e.g., after every "git" command!
+  
+
+We want to keep the "Master branch" clean and up-to-date with the StackClub repo, and
+develop in one's own "Development branch"; e.g., within the StackClub directory:
+  git checkout -b project/helloworld/douglasleetucker
+  git branch       # Confirm you are now working on that branch
+
+
+When ready, push updates back to github:
+  git push origin project/helloworld/douglasleetucker
+  Submit a pull request via webpage:
+     https://github.com/LSSTScienceCollaborations/StackClub/pulls
+
+
+* * * FOR DETAILS, SEE 'Example workflow, from "git clone" to merge with remote repo...' SECTION BELOW. * * *
+
+
+
+-----------------------------------------------------------
+Example workflow, from "git clone" to merge with remote repo...
+
+See also:
+  https://github.com/LSSTScienceCollaborations/StackClub/blob/master/GettingStarted/HelloWorld.ipynb
+
+
+# Clone StackClub repo (first time) and create a new branch:
+cd notebooks
+git clone https://github.com/LSSTScienceCollaborations/StackClub.git   # first time
+cd StackClub
+git remote -v    # check remote repo
+git branch -a    # list all branches
+git status       # very useful; e.g., after every "git" command!
+git checkout -b project/helloworld/douglasleetucker   # Create a new branch and move to it...
+git branch       # Confirm you are now working on that branch
+
+#
+# * * * Make changes to the branch here. * * *
+#
+If editing a notebook:
+  1. make your changes.
+  2. check your changes work.
+  3. from menu bar:    Kernel --> Restart and Run All Cells
+         a. fix any errors
+	 b. re-do until errors are all corrected
+  4. from menu bar:    Kernel --> Restart and Clear All Cells
+  5. hit "save" in the notebook editor to make sure edits are captured.
+
+
+
+# Commit changes to local branch:
+git status
+git add <filename>      # if there is a file ("<filename>") to add...
+git commit -am "Add a fairly detailed comment on the changes you made here"  # check in changes
+git status
+
+
+# Push updates to origin on github:
+git push origin project/helloworld/douglasleetucker
+
+
+# Make "pull" request:
+Go to https://github.com/LSSTScienceCollaborations/StackClub/pulls and click on green "New pull request" button.
+
```

Comment:
I think "pull request" is possibly the most mysterious command to newbies.  A couple of lines briefly explaining what this "pull request" is, and the approval process might be helpful to newbies...